azurerm/internal/services/web/app_service.go

```diff
@@ -1735,7 +1745,15 @@ func flattenAppServiceIpRestriction(input *[]web.IPSecurityRestriction) []interf
 			if *ip == "Any" {
 				continue
 			} else {
-				restriction["ip_address"] = *ip
+				switch v.Tag {
+				case web.Default:
+					restriction["ip_address"] = *ip
+				case web.ServiceTag:
+					restriction["service_tag"] = *ip
+				default:
+					// FIXME: I want to error here if we got XffProxy. It doesn't seem possible to set in the portal
+					// but can theoretically come back from the API as defined
```

Comment:
I think we can safely ignore it?

Reply:
Ok, what I did for now was to set the `ip_address` property in the default case, this might be incorrect in the long term if this is to be supported, but doesn't change the behaviour from before.